demo-kotlin/app/src/main/java/com/example/placesdemo/programmatic_predictions/ProgrammaticAutocompleteToolbarActivity.kt

```diff
@@ -100,25 +134,36 @@ class ProgrammaticAutocompleteToolbarActivity : AppCompatActivity() {
         val layoutManager = LinearLayoutManager(this)
         recyclerView.layoutManager = layoutManager
         recyclerView.adapter = adapter
-        recyclerView .addItemDecoration(DividerItemDecoration(this, layoutManager.orientation))
+        recyclerView.addItemDecoration(DividerItemDecoration(this, layoutManager.orientation))
+        adapter.onPlaceClickListener = { geocodePlaceAndDisplay(it) }
     }
 
     /**
-     * This method demonstrates the programmatic approach to getting place predictions
+     * This method demonstrates the programmatic approach to getting place predictions. The
+     * parameters in this request are currently biased to Kolkata, India.
      *
-     * @param query the plus code query string (e.g. "MC2W+RG K")
+     * @param query the plus code query string (e.g. "GCG2+3M K")
      */
     private fun getPlacePredictions(query: String) {
+        // The value of 'bias' biases prediction results to the rectangular region provided
+        // (currently Kolkata). Modify these values to get results for another area.
```

Comment:
Also mention .setCountries as another critical parameter for delivering good/expected results.